jplag.frontend.golang/src/test/java/de/jplag/golang/GoFrontendTest.java

```diff
@@ -0,0 +1,178 @@
+package de.jplag.golang;
+
+import static de.jplag.golang.grammar.GoTokenUtils.getDummyToken;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.OptionalInt;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import java.util.stream.StreamSupport;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import de.jplag.Token;
+import de.jplag.TokenConstants;
+import de.jplag.TokenList;
+import de.jplag.TokenPrinter;
+import de.jplag.golang.grammar.GoTokenUtils;
+import de.jplag.testutils.TestErrorConsumer;
```

Comment:
pull the imports apart as you have done so far in the other classes.

Reply:
Can you explain what you mean? I let the IDE handle the imports.